source/common/http/conn_manager_impl.cc

```diff
@@ -756,6 +756,17 @@ void ConnectionManagerImpl::onDrainTimeout() {
   checkForDeferredClose(false);
 }
 
+void ConnectionManagerImpl::sendGoAwayandClose() {
+  if (!codec_ || codec_->protocol() != Protocol::Http2) {
```

Comment:
Why is it only for H/2? H/3 has the same semantics. And connection closure is applicable to H/1 connections.

Reply:
My initial goal is only to support h/2, but supporting three sgtm, done.